composer.json

```diff
@@ -66,9 +66,6 @@
             "Tests/"
```

Comment:
Please have a look at https://getcomposer.org/doc/04-schema.md#exclude-files-from-classmaps. The test are still passing, so no need for this entry.

Btw. I stumpled upon this entry in https://github.com/FriendsOfSymfony/FOSUserBundle/pull/2412